openslides_backend/action/actions/user/password_mixin.py

```diff
@@ -0,0 +1,27 @@
+import random
+from typing import Any, Dict
+
+from ...action import BaseAction
+
+
+class PasswordCreateMixin(BaseAction):
+    def set_password(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        password = instance.get("default_password", "")
+        instance["default_password"] = password
+        hashed_password = self.auth.hash(password)
+        instance["password"] = hashed_password
+        return instance
+
+    def generate_and_set_password(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        password = self.generate_password()
+        instance["default_password"] = password
+        hashed_password = self.auth.hash(password)
+        instance["password"] = hashed_password
+        return instance
```

Comment:
And this can be unified with the method above

Reply:
Done